rmgpy/data/kinetics/familyTest.py

```diff
@@ -1,22 +1,24 @@
 import unittest
-from rmgpy.data.kinetics.database import KineticsDatabase
 import os.path
-from rmgpy.molecule.group import Group
+
 from rmgpy import settings
+from rmgpy.data.kinetics.database import KineticsDatabase
+from rmgpy.data.kinetics.family import ReactionRecipe
+from rmgpy.molecule import Molecule
 ###################################################
 
 class TestFamily(unittest.TestCase):
 
-
-    def setUp(self):
+    @classmethod
```

Comment:
I don't think there's a need to tear down, since this test class only loads a `KineticsDatabase()` instance, which does not interact with the global `Database` variable. Let me know if you think otherwise though.